src/merkle/store/mod.rs

```diff
@@ -430,3 +431,45 @@ impl MerkleStore {
         }
     }
 }
+
+impl Serializable for Node {
```

Comment:
nit: I'd put a divider above this line titled something like `SERIALIZATION`.

Reply:
done